Source/RenderStreamEditor/Private/RenderStreamEditorModule.cpp

```diff
@@ -512,7 +519,8 @@ URenderStreamChannelCacheAsset* UpdateLevelChannelCache(ULevel* Level)
     FAssetRegistryModule::AssetCreated(Cache);
     const FString PackageFileName = FPackageName::LongPackageNameToFilename(CacheFolder + LevelPath, FPackageName::GetAssetPackageExtension());
     FSavePackageArgs args;
-    args.SaveFlags = EObjectFlags::RF_Public | EObjectFlags::RF_Standalone;
+    args.TopLevelFlags = EObjectFlags::RF_Public | EObjectFlags::RF_Standalone;
+    args.bSlowTask = false;
```

Comment:
SaveFlags was the wrong field to set, since both SaveFlags and TopLevelFlags are uint32 instead of the actual enum type this wasn't noticed when the code was migrated to use the FSavePackageArgs structure (which was introduced in 5.0)

Reply:
Slow task was also set correctly in 4.27 but the default value was wrong in the structure.